.github/workflows/ci.yml

```
@@ -0,0 +1,82 @@
+  contents: read
+  pull-requests: write
+  security-events: write
+
+jobs:
+  semgrep:
+    name: Run Semgrep Scan
+    runs-on: ubuntu-latest
+    # 設定此 job 失敗時仍然不阻斷整個 workflow
+    continue-on-error: true
+    outputs:
+      semgrep_results: ${{ steps.check.outputs.semgrep_results }}
```

actionlint (1.7.4)

property "check" is not defined in object type {}

(expression)